zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumPeer.java

```diff
@@ -2460,4 +2460,13 @@ boolean isLeader(long id) {
         }
     }
 
+    public int getMaxCnxns() {
```

Comment:
Same code is used in org.apache.zookeeper.server.ZooKeeperServer#getMaxCnxns. Please move this code to common place and reuse it.

Reply:
Done, thanks :)